include/quda.h

```diff
@@ -507,7 +511,7 @@ extern "C" {
 
     /** Whether to use the smeared gauge field for the Dirac operator
         for whose eigenvalues are are computing. */
-    bool use_smeared_gauge;
+    QudaBoolean use_smeared_gauge;
```

Comment:
Why the change to `QudaBoolean` from `bool`?  I intentionally used `bool` with this variable as we intend to deprecate `QudaBoolean`.

Reply:
Also I think with this addition of the `use_smeared_gauge` parameter to `QudaInvertParm` the `QudaEigParam` variant is never used now?  Can we just delete it?

Reply:
I used QudaBoolean because all the other boolean members did...it looked weird to see a difference.  But bool is fine.  The use_smeared_gauge is needed in the InvertParam  because only that gets sent to the setDiracParam functions, where it is needed.  It is ok to keep it in the EigParam too.  I think I put in a check that these two bools have to be the same.